cluster-template/phd-{{cookiecutter.cluster_slug_normalized}}-cluster/.github/.commitlintrc.mjs

```diff
@@ -0,0 +1,65 @@
+const actor = process.env.GITHUB_ACTOR || "";
+
+function isBotActor() {
+  return [
+    "actions-user",
+    "dependabot[bot]",
+  ].indexOf(actor) !== -1;
+}
+
+function isDependencyUpdateCommit(message) {
+  return message.startsWith("chore(deps):");
+}
+
+function shouldIgnoreCommit(message) {
+  return isBotActor() || isDependencyUpdateCommit(message);
+}
+
+export default {
+  extends: ["@commitlint/config-conventional"],
+  defaultIgnores: false,
```

Comment:
Nit: Just wanted to highlight that, one of the [defaults ignores](https://github.com/conventional-changelog/commitlint/blob/master/%40commitlint/is-ignored/src/defaults.ts) is merge commits. Should we add merge commits to our custom igrone commit logic below ?